packages/sdk-js/src/DidHelpers/setVerificationMethod.ts

```diff
@@ -82,3 +84,82 @@ export function setVerificationMethod(
     expectedEvents: [{ section: 'did', method: 'DidUpdated' }],
   })
 }
+
+/**
+ * Removes the verification method for the selected `verificationMethodId` and  `relationship`.
+ *
+ * Note: authentication verification method can not be removed.
+ *
+ * @param options Any {@link SharedArguments} and additional parameters.
+ * @param options.relationship The relationship for which this verification method shall be useable.
```

Comment:
It's the opposite, right? The relationship for which this VM shall be deactivated / become unusable.